src/flow-debugger.html

```diff
@@ -625,14 +625,19 @@
                     }
 
                     var tools = $('<span class="red-ui-flow-debugger-msg-tools button-group"></span>').appendTo(topRow);
-                    var stepItem = $('<button class="red-ui-button red-ui-button-small"><i class="fa fa-step-forward"></i></button>').appendTo(tools).on("click", function(evt) {
+                    var stepItem = $('<button class="red-ui-button red-ui-button-small"><i class="fa fa-step-forward"></i></button>').appendTo(tools);
+                    var stepMessage = RED.popover.tooltip(stepItem, RED._("node-red-debugger/flow-debugger:label.stepMessage"));
+                    stepItem.on("click", function(evt) {
                         evt.preventDefault();
                         evt.stopPropagation();
                         postCommand("step",{message: item.id})
                         messageList.editableList("removeItem",item);
+                        stepMessage.close();
                     });
 
-                    var deleteItem = $('<button class="red-ui-button red-ui-button-small"><i class="fa fa-trash"></i></button>').appendTo(tools).on("click", function(evt) {
+                    var deleteItem = $('<button class="red-ui-button red-ui-button-small"><i class="fa fa-trash"></i></button>').appendTo(tools);
+                    var deleteMessage = RED.popover.tooltip(deleteItem, RED._("node-red-debugger/flow-debugger:label.deleteMessage"));
```

Comment:
I didn't add these tooltips because I found deleting the message when the tooltip was shown could lead to the tooltip staying open and never closing.

There's some more work to be done in the core tooltip code to fix that. Until that is resolved, we don't want tooltips here.

Reply:
Actually - looking again, I see you call the close function on the tooltip. That is probably good enough - so ignore my previous comment.